Functions.Templates/Documentation/tableIn.md

```diff
@@ -64,25 +59,32 @@ module.exports = function (context, myQueueItem) {
 };
 ```
 
-#### Storage tables example: C# example that reads multiple table entities
+#### Storage tables example: C# example that reads multiple table entities based on a queue trigger
 
-The C# code adds a reference to the Azure Storage SDK so that the entity type can derive from `TableEntity`.
+The C# code adds a reference to Newtonsoft JSON library so that we can utilize JSON constructs to process multiple table entries. The `queuePerson` argument is bound to a queue entry while the `inputTable` is bound to an Azure Storage Table. We can specify the `filter` property so that, for example, only table results for persons older than the person in the queue are returned ( specify `"Age gt {Age}"` where `{Age}` represents the age of the person from the queue). 
 
 ```csharp
-#r "Microsoft.WindowsAzure.Storage"
-using Microsoft.WindowsAzure.Storage.Table;
+#r "Newtonsoft.Json"
+using System;
 
-public static void Run(string myQueueItem, IQueryable<Person> tableBinding, ILogger log)
+public static void Run(Person queuePerson, Newtonsoft.Json.Linq.JArray inputTable, ILogger log)
 {
-    log.LogInformation($"C# Queue trigger function processed: {myQueueItem}");
-    foreach (Person person in tableBinding.Where(p => p.PartitionKey == myQueueItem).ToList())
+    foreach (Newtonsoft.Json.Linq.JToken jToken in inputTable)
     {
-        log.LogInformation($"Name: {person.Name}");
+        Person p = Newtonsoft.Json.JsonConvert.DeserializeObject<Person>(jToken.ToString());

```

Comment:
You should be able to bind to a Person[] directly rather than JArray. Doesn't that work?

Reply:
Just tried this using both the WebJobs project and the Portal code editor. The WebJobs project is showing this an invalid binding while the Portal code is failing silently even though it compiles (I imagine that this is because of some background failure similar to the one I saw in the WebJobs project).